src/components/para_trials/counter.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import styles from "../../styles/Paraflow.module.css";
+
+interface CounterProps {
+  title: string;
+  maxCount: number;
+  minCount: number;
+  color: string;
+}
+
+const Counter = ({ title, maxCount, minCount, color }: CounterProps) => {
+  // Count variable may need to become a prop, depending on how we implement
+  const [count, setCount] = useState(0);
+
+  const incrementCount = () => {
+    if (count === maxCount) {
+      return;
+    }
+    setCount(count + 1);
+  };
+  const decrementCount = () => {
+    if (count === minCount) {
+      return;
+    }
+    setCount(count - 1);
```

Comment:
🐧 mostly personal style preference but could do 
```suggestion
    setCount(Math.max(count - 1, minCount));
```
here instead (and likewise above)